db/migrate/20181222164913_add_gift_credit_to_organization_transactions.rb

```diff
@@ -0,0 +1,6 @@
+class AddGiftCreditToOrganizationTransactions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :organization_transactions, :gift_credit, :boolean, default: false
+    add_index :organization_transactions, :gift_credit
```

Comment:
I might consider naming this column simply `gift`. This would support `gift` debits also. Probably a slim chance of this happening, but an advertiser could "gift" some money from their balance to CodeFund to distribute to open source maintainers.